book/running-wallaroo/autoscale.md

```diff
@@ -0,0 +1,83 @@
+# Autoscale
+
+Wallaroo is designed to support scale-independent development. It can adapt to changing demands by growing or shrinking to fit the available resources. You don’t have to update any code or bother with stopping your cluster and redeploying. This set of features is called “autoscale”.
+
+With autoscale enabled, you can add or remove workers from a running cluster. We call these “grow to fit” and “shrink to fit”, respectively (referring to the fact that the cluster grows or shrinks to fit your current resource requirements). To enable autoscale, you must build a Wallaroo binary using the `-D autoscale` command line argument. Autoscale is set by default when running `make` for building Machida or the example Wallaroo applications.
+
+## Grow to Fit   
+
+While a Wallaroo cluster is running, new workers can join the cluster.  To add workers to a running cluster, you must know the control channel address of one of the workers in the running cluster.  Let’s assume it’s `127.0.0.1:12500`.  The following command would allow us to add one worker to a running cluster for the `alphabet` example app:
+
+{% codetabs name="Python", type="py" -%}
+```
+machida --application-module alphabet --in 127.0.0.1:7010 \
+--out 127.0.0.1:7002 --join 127.0.0.1:12500 --name w3 \
+--ponythreads 1
+```
+{%- language name="Go", type="go" -%}
+```
+alphabet --in 127.0.0.1:7010 --out 127.0.0.1:7002 \
```

Comment:
The command lines after the first one should be indented.

Reply:
done for all these indents